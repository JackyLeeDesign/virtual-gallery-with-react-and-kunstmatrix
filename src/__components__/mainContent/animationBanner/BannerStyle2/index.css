.main_banner {
  position: relative;
  width: 100%;
  height: auto;
  z-index: -1;
  overflow: hidden;
  background: #d1a689;
  /* box-shadow: 0px 2px 6px 1px #3d3d3d; */
}

.cloud1 {
  position: absolute;
  width: 30%;
  top: 10%;
  z-index: 1;
  animation: cloud1 15s;
  animation-iteration-count: infinite;
  animation-fill-mode: forwards;
  animation-timing-function: ease;
  animation-direction: alternate;
}

.cloud2 {
  position: absolute;
  width: 25%;
  top: 20%;
  z-index: 1;
  animation: cloud2 10s;
  animation-iteration-count: infinite;
  animation-fill-mode: forwards;
  animation-timing-function: ease;
  animation-direction: alternate;
}

.cloud3 {
  position: absolute;
  width: 25%;
  top: 8%;
  z-index: 1;
  animation: cloud3 20s;
  animation-iteration-count: infinite;
  animation-fill-mode: forwards;
  animation-timing-function: ease;
  animation-direction: alternate;
}

.main_banner_img {
  position: relative;
  width: 100%;
  height: auto;
  z-index: 0;
  overflow: hidden;
  background: #c7e7fa;
}

.main_banner_img2 {
  position: absolute;
  width: 100%;
  height: auto;
  z-index: 2;
  overflow: hidden;
}

.car_pink {
  position: absolute;
  width: 6%;
  left: -8%;
  top: 65%;
  z-index: 1;
  animation: car_LtoR 20s;
  animation-iteration-count: infinite;
  animation-fill-mode: forwards;
  animation-timing-function: linear;
}

.car_blue {
  position: absolute;
  width: 6%;
  top: 66%;
  left: -8%;
  z-index: 1;
  animation: car_LtoR 15s;
  animation-iteration-count: infinite;
  animation-fill-mode: forwards;
  animation-timing-function: linear;
  animation-delay: 3s;
}

.car_orange {
  position: absolute;
  width: 6%;
  top: 66%;
  left: -8%;
  z-index: 1;
  animation: car_LtoR 25s;
  animation-iteration-count: infinite;
  animation-fill-mode: forwards;
  animation-timing-function: linear;
  animation-delay: 6s;
}

.car_white {
  position: absolute;
  width: 6%;
  top: 69%;
  left: 100%;
  z-index: 1;
  animation: car_RtoL 22s;
  animation-iteration-count: infinite;
  animation-fill-mode: forwards;
  animation-timing-function: linear;
}

.car_green {
  position: absolute;
  width: 6%;
  top: 70%;
  left: 100%;
  z-index: 1;
  animation: car_RtoL 30s;
  animation-iteration-count: infinite;
  animation-fill-mode: forwards;
  animation-timing-function: linear;
  animation-delay: 6s;
}

.blackcloud {
  position: absolute;
  width: 5%;
  z-index: 1;
  animation: blackcloud 5s;
  animation-iteration-count: infinite;
  animation-fill-mode: forwards;
  animation-timing-function: linear;
}

@keyframes blackcloud {
  0% {
    top: 8%;
    left: 5%;
    opacity: 0;
  }
  50% {
    top: 5%;
    left: 6%;
    opacity: 1;
  }
  100% {
    top: 2%;
    left: 7%;
    opacity: 0;
  }
}

@keyframes cloud1 {
  from {
    left: 5%;
  }
  to {
    left: 25%;
  }
}

@keyframes cloud2 {
  from {
    right: 28%;
  }
  to {
    right: 40%;
  }
}

@keyframes cloud3 {
  from {
    left: 60%;
  }
  to {
    left: 85%;
  }
}

@keyframes car_LtoR {
  from {
    left: -15%;
  }
  to {
    left: 100%;
  }
}

@keyframes car_RtoL {
  from {
    left: 100%;
  }
  to {
    left: -15%;
  }
}
