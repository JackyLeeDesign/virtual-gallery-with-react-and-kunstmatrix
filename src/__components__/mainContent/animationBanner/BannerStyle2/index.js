import React from "react";
import style from "./index.css";
import zindex2 from "./assets/zindex2.png";
import blackcloud from "./assets/blackcloud.png";
import cloud1 from "./assets/cloud1.png";
import cloud2 from "./assets/cloud2.png";
import cloud3 from "./assets/cloud3.png";
import car_pink from "./assets/car_pink.png";
import car_blue from "./assets/car_blue.png";
import car_orange from "./assets/car_orange.png";
import car_white from "./assets/car_white.png";
import car_green from "./assets/car_green.png";
import zindex0 from "./assets/zindex0.png";
const Banner = () => {
  return (
    <div className={style.main_banner}>
      <img className={style.main_banner_img2} src={zindex2} />
      <img className={style.blackcloud} src={blackcloud} />
      <img className={style.cloud1} src={cloud1} />
      <img className={style.cloud2} src={cloud2} />
      <img className={style.cloud3} src={cloud3} />
      <img className={style.car_pink} src={car_pink} />
      <img className={style.car_blue} src={car_blue} />
      <img className={style.car_orange} src={car_orange} />
      <img className={style.car_white} src={car_white} />
      <img className={style.car_green} src={car_green} />
      <img className={style.main_banner_img} src={zindex0} />
    </div>
  );
};
export default Banner;
