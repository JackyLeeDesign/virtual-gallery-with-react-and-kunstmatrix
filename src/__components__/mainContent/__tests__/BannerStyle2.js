import { configure } from "enzyme";
import renderer from "react-test-renderer";
import Adapter from "enzyme-adapter-react-16";
configure({ adapter: new Adapter() });
import React from "react";
import BannerStyle2 from "../animationBanner/BannerStyle2";
describe("BannerStyle2:UI Test", () => {
  it("UI test", () => {
    const component = renderer.create(<BannerStyle2 />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
