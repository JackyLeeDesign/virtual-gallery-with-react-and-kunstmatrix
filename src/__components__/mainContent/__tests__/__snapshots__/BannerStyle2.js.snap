// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BannerStyle2:UI Test UI test 1`] = `
<div
  className="main_banner"
>
  <img
    className="main_banner_img2"
    src="zindex2.png"
  />
  <img
    className="blackcloud"
    src="blackcloud.png"
  />
  <img
    className="cloud1"
    src="cloud1.png"
  />
  <img
    className="cloud2"
    src="cloud2.png"
  />
  <img
    className="cloud3"
    src="cloud3.png"
  />
  <img
    className="car_pink"
    src="car_pink.png"
  />
  <img
    className="car_blue"
    src="car_blue.png"
  />
  <img
    className="car_orange"
    src="car_orange.png"
  />
  <img
    className="car_white"
    src="car_white.png"
  />
  <img
    className="car_green"
    src="car_green.png"
  />
  <img
    className="main_banner_img"
    src="zindex0.png"
  />
</div>
`;
